fiftyone/core/threed/pointcloud.py

```diff
@@ -28,8 +29,11 @@ class PointCloud(Object3D):
             :class:`fiftyone.core.threed.Scene` can have at most one asset
             flagged for orthographic projection. Defaults to ``False``. If
             multiple assets are flagged, the first one will be chosen
-        **kwargs: keyword arguments for the
-            :class:`fiftyone.core.threed.Object3D` parent class
+                name: the name of the object
```

Comment:
`name` is defined twice in this docstring. I think this one needs to be deleted (also indentation is borked)